package.json

```diff
@@ -40,7 +39,7 @@
     "portfinder": "~1.0.10",
     "request": "~2.87.0",
     "requestretry": "~2.0.2",
-    "sauce-connect-launcher": "github:saucelabs/sauce-connect-launcher#v1.3.1",
+    "saucelabs": "^4.6.2",
```

Comment:
To be safer, can this be changed to `~`?

Reply:
Updated